addons/website_slides_forum/models/forum.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Forum(models.Model):
+    _inherit = 'forum.forum'
+
+    slide_channel_id = fields.Many2one('slide.channel', 'Course')
```

Comment:
Main issue: handle privaty / security as I don't think forums have real security checks. This should be clarified.